specs/filebeat.yml

```diff
@@ -0,0 +1,175 @@
+$schema: "file://../schemas/component.spec.yml"
```

Comment:
I don't see this file in the PR, I thought you reverted it to json spec?

Reply:
I removed it completely as it doesn't apply to YAML spec anyway. Forgot to remove the reference here.